react/src/components/Checkout.js

```diff
@@ -72,18 +72,30 @@ class Checkout extends Component {
     this.setState({...this.state,loading: true});
 
     let response = await this.checkout(cart)
-    if (!response.ok) {
-      Sentry.captureException(new Error(response.status + " - " + (response.statusText || "Internal Server Error")))
-    }
-
-    this.setState({...this.state,loading: false});
-    transaction.finish();
-
-    if (!response.ok) {
-      this.props.history.push('/error', {"error": "errorInfo"})
-    } else {
-      this.props.history.push('/complete', {"complete": "completeOrderInfo"})
-    }
+
+    // try{
+      
+      this.setState({...this.state,loading: false});
+      transaction.finish(Date.now()+5000);
```

Comment:
Do you remember what was this about?

Reply:
`Date.now()+5000`

Reply:
I don't, but highly recommend removing any date/time manipulations with transactions. Risky business, leads to unwanted behaviors (broken traces, traces that don't look right).

Responding super late here so I think this is being ignored already. @realkosty